siliconcompiler/core.py

```diff
@@ -4134,12 +4134,13 @@ def run(self):
                 # Use the default config file path.
                 cfg_dir = os.path.join(Path.home(), '.sc')
                 cfg_file = os.path.join(cfg_dir, 'credentials')
-            if (not os.path.isdir(cfg_dir)) or (not os.path.isfile(cfg_file)):
-                self.error('Could not find remote server configuration - '
-                    'please run "sc-configure" and enter your server address and '
-                    'credentials.', fatal=True)
-            with open(cfg_file, 'r') as cfgf:
-                self.status['remote_cfg'] = json.loads(cfgf.read())
+            if os.path.isdir(cfg_dir) and os.path.isfile(cfg_file):
+                with open(cfg_file, 'r') as cfgf:
+                    self.status['remote_cfg'] = json.loads(cfgf.read())
+            else:
+                self.logger.warning('Could not find remote server configuration: defaulting to '
+                                    'https://server.siliconcompiler.com')
+                self.status['remote_cfg'] = { "address": "https://server.siliconcompiler.com" }
```

Comment:
Can we store this url in the meta data and import it so we only need to track it one place?

Reply:
Yes, good idea.